build.gradle.kts

```diff
@@ -255,8 +249,8 @@ fun Project.configurePublishing() {
         }
       }
 
-      withType<MavenPublication>() {
-        setDefaultPomFields()
+      withType<MavenPublication> {
+        setDefaultPomFields(this)
```

Comment:
The scoping was wrong here and the pom properties were always read from the root `gradle.properties`